openupgrade_scripts/scripts/utm/16.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,13 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(env.cr, """ALTER TABLE utm_campaign ADD title jsonb;""")
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE utm_campaign
+        SET title = jsonb_build_object('en_US', to_jsonb(name));
```

Comment:
This is losing possible translations. Isn't valid the method you provided in openupgradelib? https://github.com/OCA/openupgradelib/pull/306

Reply:
You're right, I missed translations.
The method I provided is useless here because translations are already migrated in OpenUpgrade pre-migration script of base module.
Hence when starting migrating utm module, `name` column is a jsonb column containing all the translations. Hence it suffices to copy this column to populate the new `title` column.